app/player/player.controller.js

```diff
@@ -3,12 +3,15 @@
 angular.module('app.player')
```

Comment:
About previous images, did you think to use some CSSs to have background color ?